### PR TITLE
Setup the repository

### PR DESCRIPTION
**Pull request by michplunkett**
## Describe your changes
Setting up the repository so that we can begin working.

## Non-obvious technical information
N/A

## Checklist before requesting a review
- [x] The code runs successfully.

```
(poetry-python-project-py3.11) michaelp@MacBook-Air-3 supreme-court-ml-predictions % make format
black "supreme_court_predictions"/ test/
All done! ✨ 🍰 ✨
11 files left unchanged.
(poetry-python-project-py3.11) michaelp@MacBook-Air-3 supreme-court-ml-predictions % make lint
pylint "supreme_court_predictions"/ test/
ma************* Module supreme_court_predictions.__main__
supreme_court_predictions/__main__.py:1:0: C0103: Module name "__main__" doesn't conform to '^(_?[a-z][a-z0-9_]*|__init__)$' pattern (invalid-name)

-----------------------------------
Your code has been rated at 9.09/10

make: *** [lint] Error 16
(poetry-python-project-py3.11) michaelp@MacBook-Air-3 supreme-court-ml-predictions % make run
python -m "supreme_court_predictions"
oh, what up?
(poetry-python-project-py3.11) michaelp@MacBook-Air-3 supreme-court-ml-predictions % make get-data
python -m "supreme_court_predictions" --get-data
oh, what up?
getting that data
(poetry-python-project-py3.11) michaelp@MacBook-Air-3 supreme-court-ml-predictions % 
```
- [x] `make format` and `make lint` have been run and the output has been addressed.

### CODE REVIEW

File: supreme_court_predictions/api/convokit/client.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+"""
+This file serves as the client for convokit.
+"""
+
+
+def get_data():
```
Comment (michplunkett) [inline supreme_court_predictions/api/convokit/client.py:6]: @chanteriam, here's the file to add getting the data.

File: Makefile [hunk 2]
```diff
@@ -21,4 +21,8 @@ test-and-fail:
 
 .PHONY: run
 run:
-	python -m project
+	python -m ${BASEDIR}
+
+.PHONY: get-data
+get-data:
+	python -m ${BASEDIR} --get-data
```
Comment (michplunkett) [inline Makefile:28]: @chanteriam, here's the command that will download the data.

File: supreme_court_predictions/api/convokit/client.py (hunk 1 above)
Comment (michplunkett) [reply supreme_court_predictions/api/convokit/client.py:6]: Ideally, we'd download it and then unzip it in the `data/convokit` folder.

File: .gitignore [hunk 3]
```diff
@@ -3,6 +3,12 @@
 .vscode
 **/.DS_Store
 
+# Data folders
+/supreme_court_predictions/data/convokit/*
+!/supreme_court_predictions/data/convokit/README.md
+/supreme_court_predictions/data/clean_convokit/*
+!/supreme_court_predictions/data/clean_convokit/README.md
```
Comment (michplunkett) [inline .gitignore:10]: `.gitignore` things for the data folders.

File: supreme_court_predictions/api/convokit/client.py (hunk 1 above)
Comment (chanteriam) [reply supreme_court_predictions/api/convokit/client.py:6]: With this function, do I need to keep the main() function? @michplunkett

File: .gitignore (hunk 3 above)
Comment (chanteriam) [reply .gitignore:10]: I don't think the data is proprietary, are we just doing this bc it's large?

Comment (michplunkett) [reply .gitignore:10]: Yeah, it's just because it's a large amount of data.

File: supreme_court_predictions/api/convokit/client.py (hunk 1 above)
Comment (michplunkett) [reply supreme_court_predictions/api/convokit/client.py:6]: You shouldn't need it, no.

File: pyproject.toml [hunk 4]
```diff
@@ -13,6 +13,7 @@ black = "^23.1.0"
 requests = "^2.28.2"
 pytest = "^7.2.2"
 responses = "^0.23.1"
+convokit = "^2.5.3"
```
Comment (michplunkett) [inline pyproject.toml:16]: Added `convokit` so we could download the corpus.

**Pull request merged**